cmd/istioctl/main.go

```diff
@@ -119,7 +138,12 @@ var (
 
 	putCmd = &cobra.Command{
 		Use:   "replace",
-		Short: "Replace policies and rules",
+		Short: "Replace existing policies and rules with another from a file",
```

Comment:
Agreed. This was copied from [here](https://istio.io/docs/reference/istioctl.html#examples-of-common-operations).